lib/app/utils/hl7_validator.rb

```diff
@@ -3,6 +3,11 @@
 module Inferno
   # A validator that calls out to the HL7 validator API
   class HL7Validator
+    DETAILS_FILTER = [
+      %r{Sub-extension url 'introspect' is not defined by the Extension http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris},
+      %r{Sub-extension url 'revoke' is not defined by the Extension http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris},
+      /URL value .* does not resolve/
```

Comment:
Why did you use `/.../` here and `%r{}` above?  I think you're trying to do a regex in both and being consistent would be less confusing.

Reply:
I think it's because the URL contains `/` characters

Reply:
rubocop prefers that. It won't let me change them consistently.

Reply:
yeah looks like rubycop requires `%r{}` if there is a `\`, but requires `\` if there isn't.  I disagree with that!  Just always allow `%r{}`.